File: app/src/main/java/ru/cj264/geekbrains/android_intro/homework/ui/MainActivity.java
package ru.cj264.geekbrains.android_intro.homework.ui;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import ru.cj264.geekbrains.android_intro.homework.R;
import ru.cj264.geekbrains.android_intro.homework.ui.notes_list.NotesListFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.frame_app, new NotesListFragment(), NotesListFragment.TAG)
                    .commit();
        }
    }
}

File: app/src/main/java/ru/cj264/geekbrains/android_intro/homework/ui/MainActivity.kt
package ru.cj264.geekbrains.android_intro.homework.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import ru.cj264.geekbrains.android_intro.homework.R
import ru.cj264.geekbrains.android_intro.homework.ui.notes_list.NotesListFragment

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.frame_app, NotesListFragment(), NotesListFragment.TAG)
                    .commit()
        }
    }
}